### PR TITLE
Restore the Moped database migration action

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
Oops, [we deleted our database CI](https://github.com/cityofaustin/atd-moped/pull/781/files#diff-7b8f8f0b9f093c1dd2b47f26b26ea1508ce1d97f8485ff5a8fc9e1d088e380eb).

## Testing
I don't believe it will be possible to test this until we merge this PR.

### CODE REVIEW

**Comment by johnclary** [comment]
We're currently missing these migrations in main:
```
1662751229997  create_table_public_moped_tags                                                    Present        Not Present
1662752947833  create_table_public_moped_proj_tags                                               Present        Not Present
1662754037000  insert_moped_tags                                                                 Present        Not Present
1663016156653  pdd_new_milestones                                                                Present        Not Present
1663016172738  pdd_update_existing_milestones                                                    Present        Not Present
```

I think we have two options:
1. Merge this PR—the CI should apply all the missing migrations
2. Manually apply the missing migrations to staging before we merge this PR

File: .github/workflows/atd_moped_database.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#
+# Builds Cognito Triggers for Moped Authentication
+#
```
Comment (chiaberry) [inline .github/workflows/atd_moped_database.yml:3]: can this be updated to reflect what the script does?

Comment (johnclary) [reply .github/workflows/atd_moped_database.yml]: yes thank you ✅

**Review by chiaberry — APPROVED** [review]

File: .github/workflows/atd_moped_database.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# Applies database migrations to staging (main) and production
```
Comment (mddilley) [inline .github/workflows/atd_moped_database.yml:2]: 🙏

**Review by mddilley — APPROVED** [review]

**Review by frankhereford — APPROVED** [review]
Thank you so much for this PR and catching and providing a solution to the oversight in my PR from yesterday!

I like route 1 from John's question. I think that it has not attempted to run these migrations, and it should be able to catch itself up. If there is a problem, then doing step-by-step migrations may be needed, which is #2.

**Comment by johnclary** [comment]
👍 here we go!

**Comment by johnclary** [comment]
Added to the v1.8 RC [here](https://github.com/cityofaustin/atd-moped/pull/787).

**Pull request merged**